### PR TITLE
Improve API Docs

### PR DESCRIPTION
**Pull request by kalaiyarasiganeshalingam**
## Purpose
$Subject
## Checklist
- [ ] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: xmldata-ballerina/Module.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Module Overview
+
+This module provides functions to perform the conversion between `xml` and `json`. It provides APIs to convert a natural representation of data in XML into a natural representation of data in JSON and vice-versa.
```
Comment (MadhukaHarith92) [inline xmldata-ballerina/Module.md:3]:
```suggestion
This module provides functions to perform conversions between `xml` and `json`. It provides APIs to convert natural representations of data in XML into natural representations of data in JSON and vice-versa.
```

File: xmldata-ballerina/Module.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+## Module Overview
```
Comment (daneshk) [inline xmldata-ballerina/Module.md:1]:
```suggestion
## Overview
```

File: xmldata-ballerina/Package.md [hunk 3]
```diff
@@ -1,3 +1,15 @@
-## Package overview
+## Package Overview
 
-This package provides functions to perform the conversion between `xml` and `json`. It provides APIs to convert a natural representation of data in XML into a natural representation of data in JSON and vice-versa.
+The XmlData library is one of the standard library packages of the<a target="_blank" href="https://ballerina.io/"> Ballerina </a> language.
+
+This module provides functions to perform the conversion between `xml` and `json`. It provides APIs to convert a natural representation of data in XML into a natural representation of data in JSON and vice-versa.
```
Comment (MadhukaHarith92) [inline xmldata-ballerina/Package.md:5]:
```suggestion
This module provides functions to perform conversions between `xml` and `json`. It provides APIs to convert natural representations of data in XML into natural representations of data in JSON and vice-versa.
```

File: xmldata-ballerina/xmldata.bal [hunk 4]
```diff
@@ -32,14 +32,14 @@ public type JsonOptions record {
 #     name: "John",
 #     age: 30
 # };
-# xml|Error xmlValue = xmldata:fromJson(data);
+# xml|xmldata:Error xmlValue = xmldata:fromJson(data);
```
Comment (MadhukaHarith92) [inline xmldata-ballerina/xmldata.bal:35]: Shall we use the `check` expression here?

File: xmldata-ballerina/xmldata.bal [hunk 5]
```diff
@@ -55,12 +55,12 @@ public type XmlOptions record {
 # Converts an XML object to its JSON representation.
 # ```ballerina
 # xml xmlValue = xml `<!-- outer comment -->` + xml `<name>supun</name>`;
-# json|Error jsonValue = toJson(xmlValue);
+# json|xmldata:Error jsonValue = toJson(xmlValue);
```
Comment (MadhukaHarith92) [inline xmldata-ballerina/xmldata.bal:58]: Shall we use the `check` expression here?

File: README.md [hunk 6]
```diff
@@ -11,7 +11,9 @@ The XmlData library is one of the standard library packages of the<a target="_bl
 
 This package provides functions to perform the conversion between `xml` and `json`. It provides APIs to convert a natural representation of data in XML into a natural representation of data in JSON and vice-versa.
 
-For more information go to [The XmlData PACKAGE](https://ballerina.io/learn/api-docs/ballerina/xmldata/).
+For more information on all the operations supported by this package, go to [The XmlData Package](https://docs.central.ballerina.io/ballerina/xmldata/latest).
```
Comment (praneesha) [inline README.md:14]:
```suggestion
For more information on all the operations supported by this package, go to the [`XmlData` package](https://docs.central.ballerina.io/ballerina/xmldata/latest).
```

File: xmldata-ballerina/Package.md [hunk 7]
```diff
@@ -1,3 +1,15 @@
-## Package overview
+## Package Overview
 
-This package provides functions to perform the conversion between `xml` and `json`. It provides APIs to convert a natural representation of data in XML into a natural representation of data in JSON and vice-versa.
+The XmlData library is one of the standard library packages of the<a target="_blank" href="https://ballerina.io/"> Ballerina </a> language.
```
Comment (praneesha) [inline xmldata-ballerina/Package.md:3]:
```suggestion
The `XmlData` library is one of the standard library packages of the<a target="_blank" href="https://ballerina.io/"> Ballerina </a> language.
```

**Review by praneesha — APPROVED** [review]

**Review by MadhukaHarith92 — APPROVED** [review]
LGTM

**Pull request merged**